frontend/src/helpers/budgetLines.helpers.js

```diff
@@ -127,7 +142,20 @@ export const canLabel = (budgetLine) =>
 
 /**
  * Returns display label of a budget line.
- * @param {Object} budgetLine - The budget line to get the BLI label from.
+ * @param {BudgetLine} budgetLine - The budget line to get the BLI label from.
  * @returns {string} The BLI label of the budget line.
  */
 export const BLILabel = (budgetLine) => (isBLIPermanent(budgetLine) ? budgetLine?.id : "TBD");
+
+/**
+ * Returns whether the given budget line is editable by status.
+ * @param {BudgetLine} budgetLine - The budget line to check.
+ * @returns {boolean} Whether the budget line is editable by status.
+ **/
+export const isBudgetLineEditableByStatus = (budgetLine) => {
+    const isBudgetLineDraft = budgetLine?.status === BLI_STATUS.DRAFT;
+    const isBudgetLinePlanned = budgetLine?.status === BLI_STATUS.PLANNED;
+    const isBudgetLineInReview = budgetLine?.in_review;
```

Comment:
removed EXECUTING budget lines since we only want to allow editing on DRAFT and PLANNED